packages/types/src/types.ts

```diff
@@ -7,3 +9,10 @@ export {ts as allForks} from "./allForks";
 
 /** Common non-spec type to represent roots as strings */
 export type RootHex = string;
+
+/** This type helps response to beacon_block_by_range and beacon_block_by_root more efficiently */
+export type P2pBlockResponse = {
```

Comment:
Since this is restricted to ReqResp only and not gossip you can rename to `ReqRespBlockResponse`